packages/openapi-typescript/src/types.ts

```diff
@@ -467,6 +467,11 @@ export type SchemaObject = {
   | {}
 );
 
+export interface TransformObject {
```

Comment:
👍 This is great.

We can probably extend this pretty far before having to change this API.